kibana-reports/server/routes/utils/constants.ts

```diff
@@ -45,3 +45,5 @@ export enum SCHEDULE_TYPE {
 export enum DATA_REPORT_CONFIG {
   excelDateFormat = 'MM/DD/YYYY h:mm:ss a',
 }
+
+export const DEFAULT_MAX_SIZE = 10000;
```

Comment:
Why 10000, any reference you cant provide?

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/6.8/search-request-from-size.html